src/other/utils.js

```diff
@@ -1,6 +1,4 @@
-const angular = require('angular-fix');
-
-export default {formlyEval, getFieldId, reverseDeepMerge, findByNodeName};
```

Comment:
I can understand your preference, but just so you have some context, here's why I put exports at the bottom:

1. More intuitive mental model for structure:
    - import stuff
    - do stuff with imports and declare new stuff
    - export stuff
2. Avoid hoisting issues.  If you are simply exporting named functions, you wouldn't run into this, but if you export declared variables, you are hosed.

I'm curious as to what the larger community style is (maybe there is room in some kind of style guide for this kind of thing).

However, all that said, I respect your existing style and will change to fit that!